src/primo/browzine-primo-adapter.js

```diff
@@ -192,9 +192,11 @@ browzine.primo = (function() {
       browzineWebLinkText = browzine.articleBrowZineWebLinkText || browzine.primoArticleBrowZineWebLinkText || "View Issue Contents";
     }
 
-    var template = "<div class='browzine'>" +
+    var template = "<div class='browzine' style='line-height: 1.4em;'>" +
                       "<a class='browzine-web-link' href='{browzineWebLink}' target='_blank' title='{browzineWebLinkText} in BrowZine'>" +
-                          "<img src='{bookIcon}' class='browzine-book-icon'/> {browzineWebLinkText}" +
+                          "<img src='{bookIcon}' class='browzine-book-icon' style='margin-bottom: -1px; margin-right: 2.5px;' aria-hidden='true'/> " +
+                          "<span class='browzine-web-link-text'>{browzineWebLinkText}</span> " +
+                          "<md-icon md-svg-icon='primo-ui:open-in-new' class='md-primoExplore-theme' aria-hidden='true' style='height: 15px; width: 15px; min-height: 15px; min-width: 15px; margin-top: -2px;'><svg width='100%' height='100%' viewBox='0 0 24 24' y='504' xmlns='http://www.w3.org/2000/svg' fit='' preserveAspectRatio='xMidYMid meet' focusable='false'><path d='M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z'></path></svg></md-icon>" +
```

Comment:
Hey @tiandavis ,
Glad things are looking good and John could test them out!

One question - what is this gigantic `<path d='...'>` tag? 😮 

Could we maybe name that bit of HTML into a variable to give it some context as to what it is? I don't know what `M14,3V5H17...` is, but it makes me wonder if that's important, or if we can even delete it altogether. A little context with a variable name might be helpful for future maintenance questions.

One idea:
```
var bookIconImgTag = "<img src='{bookIcon}' class='browzine-book-icon' style='margin-bottom: -1px; margin-right: 2.5px;' aria-hidden='true'/> ";
var browzineWebLink =  "<span class='browzine-web-link-text'>{browzineWebLinkText}</span> ";
var primoExternalLinkWithDrawingCoordsButton = "<md-icon md-svg-icon='primo-ui:open-in-new' class='md-primoExplore-theme' aria-hidden='true' style='height: 15px; width: 15px; min-height: 15px; min-width: 15px; margin-top: -2px;'><svg width='100%' height='100%' viewBox='0 0 24 24' y='504' xmlns='http://www.w3.org/2000/svg' fit='' preserveAspectRatio='xMidYMid meet' focusable='false'><path d='M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z'></path></svg></md-icon>";

var template = "<div class='browzine' ...>" + 
               "<a class='browzine-web-link' href='{browzineWebLink}' target='_blank' title='{browzineWebLinkText} in BrowZine'>" +
               bookIconImgTag +
               browzineWebLink +
               primoExternalLinkWithDrawingCoordsButton +
               "</a>" +
               "</div>";
```

What do you think Tian?

Thanks!